shell/src/main/java/alluxio/shell/command/CopyFromLocalCommand.java

```diff
@@ -62,7 +63,7 @@ protected int getNumOfArgs() {
   }
 
   @Override
-  public void run(CommandLine cl) throws IOException {
+  public void run(CommandLine cl) throws AlluxioException, IOException {
```

Comment:
hm, I think you make a good point that we could consolidate `copyFromLocal`, `copyToLocal` and `cp` into a single command, which uses `file://` to identify local paths (and non-schemed paths are assumed to be alluxio paths).

I would prefer to do that in a separate PR though